backend/backend/api/routes.py

```diff
@@ -293,6 +295,11 @@ async def github_comment(repo, pr_number, request):
     return response
 
 
+async def github_pr(repo, pr_number):
+    response = await review_github_pr(Context(), repo, int(pr_number))
+    return response
```

Comment:
The function `review_github_pr` should be called with the correct type for the `pr_number` argument. It's currently being passed an integer, but it might expect a string or other type.